Lib/uuid.py

```diff
@@ -427,7 +429,7 @@ def _find_mac_near_keyword(command, args, keywords, get_word_index):
             if words[i] in keywords:
                 try:
                     word = words[get_word_index(i)]
-                    mac = int(word.replace(_MAC_DELIM, b''), 16)
+                    mac = _parse_mac(word)
                 except (ValueError, IndexError):
```

Comment:
```suggestion
                except IndexError:
```

_parse_mac should not raise `ValueError`, but returns `None` on invalid values.

Reply:
Updated.